### ISSUE TITLE
Custom admin templates do not display help text

### ISSUE THREAD

**Issue by fatboystring** [omni-digital/omni-forms#11]
Admin form help text is not displayed when our custom admin templates are in use.

https://github.com/omni-digital/omni-forms/blob/master/omniforms/templates/admin/omniforms/omnimodelform/related_form.html

**Comment by pawelad** [comment]
I already commited a fix for this in my local fork, but I was also thinking about using the `Model` values of `help_text` (and possibly other such as `default`, `required` and so on) to be the default (and by that I mean initial) values when adding that field to OmniForm.

Let me know what you think about it - as far as I know not doing that could be done on purpose.

**Comment by fatboystring** [comment]
Hi Pawel.  Sorry but I'm not sure I completely understand.  Is there any chance you could rephrase please?

**Comment by pawelad** [comment]
Sorry about that, I see how what I wrote can be confusing.

The problem menton here will be definitely solved by #9, but while fixing it I found another thing that I think is missing and wanted to note it somewhere (in retrospect maybe here wasn't the best place for that ; ).

What I meant was that when adding a field to a `OmniForm` we are given an opportunity to set field's 'options' such as 'help_text', 'required' and so on, but the initial/default values for them aren't taken from the field initialization.
For example when adding `username` field to `User` model `OmiForm`, I think it would be great to see `help_text` filled with Django's default 'Required. 150 characters or fewer. Letters, digits and @/./+/-/_ only.' that could be then edited. The same could be done to other mirrored field options.
